src/include/platform/CHIPDeviceEvent.h

```diff
@@ -366,8 +366,8 @@ struct ChipDeviceEvent final
         LambdaBridge LambdaEvent;
         struct
         {
-            AsyncWorkFunct WorkFunct;
             intptr_t Arg;
+            AsyncWorkFunct WorkFunct;
```

Comment:
Why was it needed to change order here?

Reply:
At one point I added another argument and tweaked the order in the struct, but then changed the mechanism again and that argument went away. I'll restore the original order. In general I uploaded this PR to make a backup and "early access review" but it still needed to be gone over by me, hence little things like this, extra debug logs, etc. as I was working on it. I will get all those.

Reply:
Order restored.